src/D2L.Bmx/BmxConfigProvider.cs

```diff
@@ -53,6 +54,28 @@ internal class BmxConfigProvider : IBmxConfigProvider {
 		);
 	}
 
+	public void SaveConfiguration( BmxConfig config ) {
+		using StreamWriter writer = new StreamWriter( ConfigFileName(), false );
```

Comment:
avoid redundant type name, and please name the literal argument `false`.
```suggestion
		using var writer = new StreamWriter( ConfigFileName(), false );
```